### DISCUSSION TITLE
How to debug a black screen

### DISCUSSION THREAD

**Discussion by vesperay** [hajimehoshi/ebiten#2739] [Q&A]
I have a user who is reporting a black screen after pressing a button. I can't think of any game logic that would paint the screen black. There is also code to recover from panics in `update` and `draw` loop and log it but nothing is logged. The screen is black and music continues to play in the background. Any suggestions for what can be done to capture more information about this problem?

**Comment by hajimehoshi** [comment]
I would like to know more details of the user's environment like OS and graphic card.

**Reply by vesperay** [reply]
Windows 10, AMD Ryzen 7 5800H

**Reply by hajimehoshi** [reply]
Also I would like to see your game code

**Reply by vesperay** [reply]
Turns out there was a bug in the game logic that was causing a panic but io.MultiWriter wasn't logging it. So no issue on ebitengine side.
